### ISSUE TITLE
Vibrato

### ISSUE THREAD

**Issue by mogenslundholm** [w3c/musicxml#174]
I believe there should be an element &quot;vibrato&quot; also:
This would be a change in note.mod:

&lt;!ELEMENT ornaments
         	(((trill-mark | turn | delayed-turn | inverted-turn |
    	   delayed-inverted-turn | vertical-turn | shake |
    	   wavy-line | mordent | inverted-mordent | schleifer |
&#43;              vibrato |          
    	   tremolo | other-ornament), accidental-mark*)*)&gt;

The question is: Should there also be a special &quot;guitar-vibrato&quot;?

**Comment by mdgood** [comment]
Thanks for this suggestion, Mogens. How would the appearance of this element differ from the existing wavy-line element?

If the distinction is in playback, perhaps we could add something specific to vibrato to the `<play>` element?

**Comment by mdgood** [comment]
Also, might this duplicate part of what is already included in issue #103 for adding more semantics to guitar notation?

**Comment by mogenslundholm** [comment]
My point of view is the player. This will never be perfect and different instruments and voices have different ways to make vibrato. The terms: Vibrato, Tremolo, Mordent and Trill are also used incorrent by many people. (according to books about music). 
Unfortunately I don't know much about graphic and what changes a new symbol will need concerning the graphic appearance.

I think that the simplest solution is OK. May look like tremolo. Or trill. Playing it should be a standard solution for all instruments. 

An italic V plus wavy-line could be fine as shown in this document:
This document says: "SMuFL contains a large variety of vibrato wiggles and ..."
http://lilypondblog.org/wp-content/uploads/2014/01/wiggles.pdf
http://lilypondblog.org/2014/02/feta-and-bravura/

Though this item has been discussed by guitarists it seems to be a general musical term. And not only a &lt;play&gt; - element either.

**Comment by mogenslundholm** [comment]
Would the best place for vibrato be in the group with "Scoop", "Doit" and  "Fall" etc. ?
(Like shown on the picture - vibrato added in the end)
![vibrato](https://cloud.githubusercontent.com/assets/3636047/26528008/c6208f26-43a0-11e7-9381-199594200252.jpg)
/Mogens

**Comment by Tails** [comment]
Agreed, MusicXML is missing a vibrato element, which would fit in the <articulations> section. It would be nice if the vibrato element would support both a "regular" as a "wide" vibrato. The "wavy line" is not enough to store the semantics of a vibrato.

**Comment by jsawruk** [comment]
I think there are two issues here: notation and playback. I think that vibrato should be treated like trills because a) both have multiple notations (trill-flat, trill-sharp, normal vibrato, wide vibrato, etc.) and b) those notations affect playback. After considering this, I think it would be best to place it within `<ornaments>`.

@mdgood: I agree, this is part of #103.
